playwright-e2e/pages/family-history.ts

```diff
@@ -0,0 +1,141 @@
+import { expect, Page } from '@playwright/test';
+import PageBase from 'pages/page-base';
+import { booleanToYesOrNo } from 'utils/test-utils';
+import { CancerSelector } from './cancer-selector';
+
+
+/**
+ * CMI's family history
+ */
+export class FamilyHistory {
+
+  private readonly page:Page;
+
+/*
+
+await page.getByText('Understanding the history of cancer in someone\'s biological family can help rese').click();
```

Comment:
curiosity question - is this supposed to go inside a function? Or is it to be used to try out later?

Reply:
Ah, I had left this when I was refactoring some generated code.  Will prune!